server/src/main/java/org/apache/cloudstack/backup/BackupManagerImpl.java

```diff
@@ -884,6 +889,18 @@ private void checkStatusOfCurrentlyExecutingBackups() {
         }
     }
 
+    @DB
+    public void cleanupBackups() {
```

Comment:
Since we're already removing backups in the VM expunge logic this is not needed. Could it cause any regression? (I understand you're ensuring that VMs without backups are removed from offering).